newcomers-tutorial.md

```diff
@@ -54,20 +54,17 @@ opt to skip this part for the time being.
   `texlive-fonts-recommended`, `fonts-lmodern`, `texlive-latex-recommended`,
   `texlive-latex-extra` - refer to your distro for available packages)
 
-### Required Bureaucracy
+### Required Accounts
 
-GHC uses its own gitlab environment, found at
-[gitlab.haskell.org](https://gitlab.haskell.org/). In order to submit merge
-requests, file issues, and participate in code reviews, you will need to create
-an account, or sign in with a GitHub account.
+In order to submit merge requests, file issues, and participate in code reviews, you will need to create a gitlab account, or sign in with GitHub account. GHC uses its own gitlab environment, found at [gitlab.haskell.org](https://gitlab.haskell.org/).
```

Comment:
Ah. Very interesting... the original intent wasn't overly clear to me! 😁 which is probably why I flipped it easily... because I didn't realise we had our own specific instance.